.github/workflows/update-attribute-dictionary-json.yml

```diff
@@ -56,3 +56,5 @@ jobs:
           base: develop
           delete-branch: true
           team-reviewers: docs-engineering
+          lables: |
+            eng
```

Comment:
i think we want content folks to review this no?

Reply:
i don't think we need to be involved unless there's a failure. similar to i18n PRs